unblob/math.py

```diff
@@ -1,3 +1,7 @@
+from typing import Callable
+
+shannon_entropy: Callable[[bytes], float]
```

Comment:
Out of curiosity, was the lack of typing reported by ruff or causing issues ? What got you to write this ?

Reply:
This is older than ruff, it is pyright LSP (editor only problem), it has bothered me for a year.

This is how importing in `processing` looks to  me without it:
![shannon_entropy-pyright](https://user-images.githubusercontent.com/1246751/232722703-0302de36-30a0-4b30-b9bf-6965adf71ffd.png)